shared/src/main/scala/mlscript/ucs/context/Scrutinee.scala

```diff
@@ -2,12 +2,9 @@ package mlscript.ucs.context
 
 import collection.mutable.{Buffer, SortedMap => MutSortedMap, SortedSet => MutSortedSet}
 import mlscript.{Lit, Loc, Var}
-import mlscript.pretyper.symbol.TypeSymbol
+import mlscript.pretyper.symbol.{ClassLikeSymbol, TypeSymbol}
 import mlscript.utils._, shorthands._
-import mlscript.DecLit
-import mlscript.IntLit
-import mlscript.StrLit
-import mlscript.UnitLit
+import mlscript.{DecLit, IntLit, StrLit, UnitLit}
```

Comment:
Instead of making all these useless imports, which are basically just noise, just make the package declaration:
```scala
package mlscript
package ucs.context
````

Reply:
Great suggestion! I've already rewritten most of the files in this way!